### ISSUE TITLE
docker "network dev-network not found"

### ISSUE THREAD

**Issue by hasahmad** [create-go-app/fiber-go-template#19]
By following the readme in the backend directory, I ran `make docker.postgres` to run pgsql db in docker, but it gave me an error `network dev-network not found`
When I checked in the Makefile, I found that it was attaching it to the network `dev-network` meaning that I had to first create docker network named `dev-network` 

So, after running `docker network create dev-network` when I ran the make command again, it was fine.

I think it's best to include this as a command in makefile like as well as mention this before running `docker.fiber` or `docker.postgres`
```bash
DEV_NET_EXISTS=`docker network ls -f name=dev-network -q | wc -l`
docker.network:
	@if [ $(DEV_NET_EXISTS) -eq 0 ]; then docker network create dev-network; fi
```

**Comment by koddr** [comment]
@hasahmad hi! Thanks for issue.

I'll fix it soon.

You can also use this way to re-check Docker network:

```makefile
docker.network:
	docker network inspect dev-network >/dev/null 2>&1 || \
	docker network create -d bridge dev-network
```

**Comment by hasahmad** [comment]
I created a pr on this with additional changes.
https://github.com/create-go-app/fiber-go-template/pull/20

**Comment by koddr** [comment]
Yep, merged #20 

Thanks!
